src/components/application_manager/test/message_helper/message_helper_test.cc

```diff
@@ -805,6 +814,310 @@ TEST_F(MessageHelperTest,
+            hmi_apis::Common_HMILevel::BACKGROUND);
+}
+
+TEST_F(MessageHelperTest, GetBCActivateAppRequestToHMISendPolicyPriorityTrue) {
+  const bool kSendPolicyPriority = true;
+  const uint32_t kCorrelationId = 10u;
+  const uint32_t kAppId = 11u;
+  const std::string kMacAdress = "mac_adress";
+  const std::string& kPolicyAppId = "policy_app_id";
+  MockApplicationSharedPtr app_shared_ptr =
+      utils::MakeShared<MockApplication>();
+  protocol_handler_test::MockSessionObserver mock_session_observer;
+  policy_test::MockPolicyHandlerInterface policy_handler;
+  application_manager_test::MockApplicationManager mock_application_manager;
+  EXPECT_CALL(policy_handler, GetUserConsentForDevice(kMacAdress))
+      .WillOnce(Return(policy::kDeviceAllowed));
+  EXPECT_CALL(*app_shared_ptr, mac_address()).WillOnce(ReturnRef(kMacAdress));
+  EXPECT_CALL(policy_handler, GetPriority(_, _)).WillOnce(Return(true));
+  EXPECT_CALL(*app_shared_ptr, policy_app_id()).WillOnce(Return(kPolicyAppId));
+  EXPECT_CALL(*app_shared_ptr, app_id()).WillOnce(Return(kAppId));
+  EXPECT_CALL(mock_application_manager, GetNextHMICorrelationID())
+      .WillOnce(Return(kCorrelationId));
+  smart_objects::SmartObjectSPtr message =
+      MessageHelper::GetBCActivateAppRequestToHMI(
+          app_shared_ptr,
+          mock_session_observer,
+          policy_handler,
+          hmi_apis::Common_HMILevel::BACKGROUND,
+          kSendPolicyPriority,
+          mock_application_manager);
+  EXPECT_EQ((*message)[strings::params][strings::function_id],
+            hmi_apis::FunctionID::BasicCommunication_ActivateApp);
+  EXPECT_EQ((*message)[strings::params][strings::message_type],
+            MessageType::kRequest);
+  EXPECT_EQ((*message)[strings::params][strings::correlation_id],
+            kCorrelationId);
+  EXPECT_EQ((*message)[strings::msg_params][strings::app_id], kAppId);
+  EXPECT_EQ((*message)[strings::msg_params][strings::activate_app_hmi_level],
+            hmi_apis::Common_HMILevel::BACKGROUND);
+}
+
+TEST_F(MessageHelperTest,
+       GetOnAppInterfaceUnregisteredNotificationToMobileCheckMessage) {
+  const int32_t kConnectionKey = 10;
+  const mobile_api::AppInterfaceUnregisteredReason::eType kReason =
+      mobile_api::AppInterfaceUnregisteredReason::APP_UNAUTHORIZED;
+  smart_objects::SmartObjectSPtr message =
+      MessageHelper::GetOnAppInterfaceUnregisteredNotificationToMobile(
+          kConnectionKey, kReason);
+  EXPECT_EQ((*message)[strings::params][strings::function_id],
+            mobile_api::FunctionID::OnAppInterfaceUnregisteredID);
+  EXPECT_EQ((*message)[strings::params][strings::message_type],
+            static_cast<int32_t>(kNotification));
+  EXPECT_EQ((*message)[strings::params][strings::connection_key],
+            kConnectionKey);
+  EXPECT_EQ((*message)[strings::msg_params][strings::reason], kReason);
+}
+
+TEST_F(MessageHelperTest, GetPriorityCodeInvalidEnum) {
```

Comment:
@okozlovlux, wrong test case naming. Should be like`<method/function name>_<data>_<expected result>`. Please check everywhere.